grants/caspershops/m3.md

```diff
@@ -0,0 +1,20 @@
+---
+Milestone: "M3",
+Status: "Completed",
+---
+<!--lang:en--> 
+### Description
+
+Payment system integration via Stripe with CSPRFiat converter
+
+
+### Acceptance Criteria
+
+Live deployment of CasperPunks Shop for initial testing and team feedback
+
+### Progress Update
+
+**Reporting Period: 04.11.2022 - 31.01.2023**
+
+Please refer to the Milestone review in the link [here](https://casper-association.atlassian.net/wiki/spaces/DropLinked/pages/284229635/Milestone+1)
```

Comment:
The Milestone review link reads "Milestone 1". Would you mind verifying, please?